### PR TITLE
fade-zoom-in.svg and style

### PR DESCRIPTION
**Pull request by mechanoid23**

### CODE REVIEW

File: src/scss/_osu_layout_builder.scss [hunk 1]
```diff
@@ -12,3 +12,8 @@
 #drupal-modal .form-item {
   width: initial !important;
 }
+
+.input-icon.fade-zoom-in {
```
Comment (emerham) [inline src/scss/_osu_layout_builder.scss]: Need to make your specificity higher, suggest using the id of #drupal-off-canvas as ID's have higher specificity.

Got it working with something like this: #drupal-off-canvas span.input-icon.fade-zoom-in

File: src/scss/_osu_layout_builder.scss [hunk 2]
```diff
@@ -12,3 +12,8 @@
...
+  background-image: url(themes/custom/madrone/assets/fade-zoom-in.svg);
```
Comment (emerham) [inline src/scss/_osu_layout_builder.scss]: Make path relative to where the final CSS file will live
background-image: url("../assets/fade-zoom-in.svg");

makes it more robust to installation source

**Review by emerham — CHANGES_REQUESTED** [review]

File: src/scss/_osu_layout_builder.scss (hunk 2 above)
Comment (mechanoid23) [reply src/scss/_osu_layout_builder.scss]: Thanks. I had a question on, it wasn't clear what the path there should be.

File: src/scss/_osu_layout_builder.scss (hunk 1 above)
Comment (mechanoid23) [reply src/scss/_osu_layout_builder.scss]: Okay, I see that ID up in the DOM. Definitely more specific.

**Review by emerham — APPROVED** [review]
Thanks for the changes. Looks good now.

**Review by Ruefa — APPROVED** [review]

**Pull request merged**